modules/tinymce/src/core/main/ts/fmt/RemoveFormat.ts

```diff
@@ -164,7 +159,7 @@ const removeNode = function (ed: Editor, node: Node, format) {
       // Wrap the block in a forcedRootBlock if we are at the root of document
       if (parentNode === dom.getRoot()) {
         if (!format.list_block || !isEq(node, format.list_block)) {
-          each(Tools.grep(node.childNodes), function (node) {
+          Arr.each(Tools.grep(node.childNodes), (node) => {
```

Comment:
Since you've swapped from `Tools.each` to `Arr.each` here, maybe also switch from `Tools.grep` to `Arr.filter`?